src/styles/blocks/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid $cardBorderColor;
+  background: $cardBgColor;
+
+  &__image-container {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__heading {
+    margin: 0 0 4px;
```

Comment:
```suggestion
    margin-bottom: 4px;
```

Reply:
h tag has default vert margins

Reply:
ok, so you can reset default styles for such tags like h1-h6, p etc
```
body, 
h2, 
p {
  margin: 0;
}
```